lib/internal/test_runner/runner.js

```diff
@@ -478,6 +500,8 @@ function watchFiles(testFiles, opts) {
       // Reset the topLevel counter
       opts.root.harness.counters.topLevel = 0;
     }
+    // TODO(pmarchini): Reset the bailed flag to rerun the tests.
+    // This must be added only when we add support for bail in watch mode.
```

Comment:
Can't we simply replace the aborted AbortController with a new one?

Reply:
I think it should work, but this PR already seems quite complex. I would prefer to implement the bail with watch mode in a second PR if this lands.
WDYT?

Reply:
I think it should work, but this PR already seems quite complex. I would prefer to implement the bail with watch mode in a second PR if this lands.
WDYT?

Reply:
Sounds good 🙂